build-system/tasks/jsify-css.js

```diff
@@ -45,6 +45,9 @@ cssnano = cssnano({
   mergeIdents: true,
   reduceIdents: false,
   zindex: false,
+  svgo: {
+    encode: true,
```

Comment:
cool! There are multiple places that have already URL encoded their SVG (either partially or fully) and this will result in double encoding. We need to decode those places. Do a search for `image/svg+xml` but places I found that need attention (plus manual testing) are:

- [ ] amp-app-banner.css
- [ ] amp-lightbox-viewer.css
- [ ] amp-sticky-ad.css in 0.1
- [ ] amp-sticky-ad.css in 1.0